src/templates/base.html

```diff
@@ -33,204 +34,188 @@
-
-<script>
-    {# From `common_settings` context processor #}
-    STORAGE_TYPE = "{{ STORAGE_TYPE }}"
-    CODALAB.state.user = {
-        id: {{ request.user.id }},
-        name: '{{ request.user.name }}',
-        username: '{{ request.user.username }}',
-        email: '{{ request.user.email }}',
-        bio: '{{ request.user.bio }}',
-        // converts python bools to js bools
-        is_staff: '{{ request.user.is_staff }}' === "True",
-        is_superuser: '{{ request.user.is_superuser }}' === "True",
-        has_competition_admin_privileges: competition => {
-            let user = CODALAB.state.user
-            if (user.is_superuser || user.is_staff) {
-                return true
-            }
-            let collabs = _.map(competition.collaborators, collab => collab.username)
-            return user.username === competition.created_by || collabs.includes(user.username)
+
+    {# Admin only user switching #}
+    <user_switch></user_switch>
+
+    <!-- Vendor -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/riot/3.13.2/riot+compiler.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.1/semantic.min.js"
+            integrity="sha256-RKNmL9+6j/3jB72OcIg8OQr91Bi4OgFPnKQOFS1O+fo=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
+    <script src="{% static "js/calendar.min.js" %}"></script>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@1.13.1/build/global/luxon.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
+
+    <!-- Ours -->
+    <script src="{% static "js/ours/client.js" %}"></script>
+    <script src="{% static "js/ours/init.js" %}"></script>
+    <script src="{% static "js/ours/riotjs_mixins.js" %}"></script>
+    <script src="{% static "js/ours/utils.js" %}"></script>
+    <script src="{% static "js/jquery.form.serializer.js" %}"></script>
+    <script src="{% static "js/Chart.bundle.js" %}"></script>
+    <script src="{% static "js/reconnecting-websocket.min.js" %}"></script>
+    <script src="{% static "generated/riot.js" %}"></script>
+
+    <script>
+        {# From `common_settings` context processor #}
+        STORAGE_TYPE = "{{ STORAGE_TYPE }}"
+        CODALAB.state.user = {
+            id: '{{ request.user.id }}',
+            name: '{{ request.user.name }}',
+            username: '{{ request.user.username }}',
+            email: '{{ request.user.email }}',
+            bio: '{{ request.user.bio }}',
         }
-    }
-    // Load in URLs or other django generated goodies
-    URLS = {
```

Comment:
Doesn't look like all of these URLs made it on to the new base.html page either

Reply:
Resolved in latest push